t/test.py

```diff
@@ -345,6 +345,14 @@ def test_strptime(self):
 
         self.assertEqual(True, dt1 == dt2)
 
+    def test_strptime(self):
```

Comment:
since there is already a method named [test_strptime](https://github.com/slashmili/python-jalali/blob/a05f8c7376ec105e46a420924c6d59265445add2/t/test.py#L340) could we use another name so it won't shadow the existing test method? maybe a more specific name for the case, for example `test_strptime_handles_alphabets_in_format`.